src/main/java/org/folio/fqm/service/FqlToSqlConverterService.java

```diff
@@ -34,17 +33,22 @@
 @Service
```

Comment:
Now that everything in here is static, should this be a `Service`? IMO it could be a util class now...

Reply:
I had the same thought, but there's still the one non-`static` method that depends on the `FqlService` :(